packages/protocol/releaseData/initializationData/release3.json

```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
-  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0.0005", "0.01", "300", "1"]
-}
\ No newline at end of file
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "4000000000000000000000000"], "ExchangeEUR"],
+  "ExchangeEUR": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "0.005", "0.0013", "300", "5"]
```

Comment:
Please explain in the PR description what's going on with StableToken being set to the zero address, and what the plan is